integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -238,7 +239,9 @@ def get_project(self, project_id: int) -> Project | None:
         else:
             return None
 
-    async def get_all_projects(self) -> typing.AsyncIterator[List[Project]]:
+    async def get_all_projects(
+        self, batch_size: int = int(os.environ.get("GITLAB_BATCH_SIZE", 100))
```

Comment:
Looked like you already passing a value for this in another place
Are we using other places 2?